src/app/pages/Unlock/planets-animation/index.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { memo, useMemo } from 'react';
+
+import clsx from 'clsx';
+
+import { Logo } from 'app/atoms/Logo';
+import { TezosNetworkLogo } from 'app/atoms/NetworkLogo';
+import { TEZOS_MAINNET_CHAIN_ID } from 'lib/temple/types';
+
+import { SUN_RADIUS } from './constants';
+import { EvmPlanetItem } from './evm-planet-item';
+import { Orbit } from './orbit';
+import styles from './PlanetsAnimation.module.css';
+import { ReactComponent as SunGlow } from './sun-glow.svg';
+import { PlanetsAnimationProps, OrbitProps } from './types';
+import { calculateBottomGapAngle } from './utils';
+
+const orbitsBase = [
+  {
+    fullRotationPeriod: 150,
+    radius: 86,
+    direction: 'clockwise' as const,
+    planets: [
+      {
+        id: 'tezos',
+        radius: 19,
+        item: <TezosNetworkLogo size={38} networkName="Tezos" chainId={TEZOS_MAINNET_CHAIN_ID} className="p-[3px]" />
+      },
+      {
+        id: 'avalanche',
+        radius: 19,
+        item: <EvmPlanetItem name="Avalanche" chainId={43114} padding="large" />
```

Comment:
pls create constants for chainIds from `EVM_DEFAULT_NETWORKS` and reuse them here